test/PhpSessionPersistenceTest.php

```diff
@@ -905,6 +907,23 @@ public function testInitializeIdRegeneratesSessionId()
         $this->assertFalse($actual->isRegenerated());
     }
 
+    public function testRegenerateWhenSessionAlreadyActiveDestroyExistingSessionFirst()
+    {
+        session_start();
+
+        $_SESSION['test'] = 'value';
+        $fileSession = (session_save_path() ?: sys_get_temp_dir()) . '/sess_' . session_id();
```

Comment:
@michalbundyra I applied fallback to `sys_get_temp_dir()` when `session_save_path()` returns empty string that was cause it tried to write session to `/` directory.